scripts/st2bootstrap-deb.sh

```diff
@@ -79,7 +172,7 @@ EHD
 
 install_st2mistral() {
   # install mistral
-  sudo apt-get install -y st2mistral
+  sudo apt-get install -y st2mistral${ST2MISTRAL_PKG_VERSION}
```

Comment:
Do we need to quote this whole string (`"st2mistral${ST2MISTRAL_PKG_VERSION}"`) to be on the safe side?